src/Admin/Views/Tools/StripeSubscriptions.cshtml

```diff
@@ -0,0 +1,265 @@
+                            </td>
+                            <td>
+                                @string.Join(",", Model.Items[i].Subscription.Items.Data.Select(product => product.Plan.Id).ToArray())
+                            </td>
+                            <td>
+                                @Model.Items[i].Subscription.CurrentPeriodEnd.ToShortDateString()
+                            </td>
+                        </tr>
+                    }
+                }
+            </tbody>
+        </table>
+    </div>
+    <nav class="d-inline-flex">
+        <ul class="pagination">
+            @if(!string.IsNullOrWhiteSpace(Model.Filter.EndingBefore))
+            {
+                <input type="hidden" asp-for="@Model.Filter.EndingBefore" value="@Model.Filter.EndingBefore">
+                <li class="page-item">
+                    <button 
+                    type="submit"
+                    class="page-link"
+                    name="action"
+                    asp-for="Action"
+                    value="@StripeSubscriptionsAction.PreviousPage"
+                    >
+                        Previous
+                    </button>
+                </li>
+            }
+            else
+            {
+                <li class="page-item disabled">
+                    <a class="page-link" href="#" tabindex="-1">Previous</a>
+                </li>
+            }
+            @if(!string.IsNullOrWhiteSpace(Model.Filter.StartingAfter))
+            {
+                <input type="hidden" asp-for="@Model.Filter.StartingAfter" value="@Model.Filter.StartingAfter">
+                <li class="page-item">
+                    <button class="page-link"
+                    type="submit"
+                    name="action"
+                    asp-for="Action"
+                    value="@StripeSubscriptionsAction.NextPage"
+                    >
+                        Next
+                    </button>
+                </li>
+            }
+            else
+            {
+                <li class="page-item disabled">
+                    <a class="page-link" href="#" tabindex="-1">Next</a>
+                </li>
+            }
+        </ul>
+        <span id="bulkActions" class="d-none ml-2">
+            <span class="d-inline-flex">
+                <button 
+                    type="submit"
+                    class="btn btn-primary mr-1"
+                    name="action"
+                    asp-for="Action"
+                    value="@StripeSubscriptionsAction.Export"
+                >
+                    Export 
+                </button>
+                <button 
+                    type="submit"
+                    class="btn btn-danger"
+                    name="action"
+                    asp-for="Action"
+                    value="@StripeSubscriptionsAction.BulkCancel"
+                    >
+                    Bulk Cancel
+                </button>
```

Comment:
It doesn't look like the Bulk Cancel button has a confirmation modal/check. I'm a little concerned that this could be dangerous because this page also shows items with a status of active. Would someone be able to accidentally bulk cancel active subscriptions?

Not sure if this is a good alternative solution, but maybe this button can be disabled if the selected status is active or not Unpaid.